Project_README.md

```diff
@@ -5,10 +5,9 @@
 
 > A single page app created with react to operate as a calculator.
 
-<!-- ![Desktop Version](./src/imgs/todo-list.png)
-![Mobile Version](./src/imgs/todo-mobile.png) -->
+[Screenshot of the UI](./src/imgs/screenshot.png)
 
-"Math Magicians" is a single page app that displays a calculator allows you to make basic operations.
+"Math Magicians" is a single page app that displays a calculator allows you to make basic operations. As of now, I have finished the second requirements, meaning the UI, without functionalities.
 
```

Comment:
- [OPTIONAL] Please kindly rename your `readme file` as `README.md` so that it is displayed as soon as one visits that particular branch.

Reply:
Thank you, I have just done that.